web/src/layout/navigation/MobileSettings.tsx

```diff
@@ -184,19 +187,25 @@ const MobileSettings = (props: Props) => {
                       </div>
                     </button>
 
-                    <button
-                      className="dropdown-item my-2"
-                      onClick={() => {
-                        setOpenSideBarStatus(false);
-                        props.setOpenSignUp(true);
-                      }}
-                      aria-label="Open sign up modal"
-                    >
-                      <div className="d-flex align-items-center">
-                        <FaEdit className="me-2" />
-                        <div>Sign up</div>
-                      </div>
-                    </button>
+                    {allowUserSignUp ? (
```

Comment:
Can you use **&&** operator? https://github.com/artifacthub/hub/blob/8cb2f3824354ea3dec3c5556e9164bf0700e5249/web/src/layout/navigation/Navbar.tsx#L67